docs/landing-zone-v2/README.md

```diff
@@ -59,17 +60,32 @@ This Landing Zone v2 assumes that the different required environments known as E
 
 ## <a name='SingleGCPorganization'></a>Single GCP organization
 
+This pattern will be ideal for most use-cases and isolation between enviornments is achieved through folder structures.
+
+In this example a Landing Zone is deployed per environment.
+
 ![img](img/single-org.png)
 
 ## <a name='MultipleGCPorganizations'></a>Multiple GCP organizations
 
+This pattern is ideal for Departments who need a higher degree of isolation between environments by using Google Cloud Organizations as the environment boundary vs using folders.
+
+Additional details about using multiple organizations can be found [here](https://cloud.google.com/resource-manager/docs/managing-multiple-orgs).
+
 ![img](img/multi-org.png)
 
 ## <a name="folderStructure"></a>Folder Structure Per Environment (Dev, Preprod, Prod)
 
+A deployed Landing Zone contains two sections the Core Landing Zone and the Client Landing Zone. Together they will deploy a structure that matches the diagram below.
+
+
 ![img](img/folder-structure.png)
 
-### Core Landing Zone Folders
+### Core Landing Zone
+
+This is made up of two pacakges `core-landing-zone` and `gatekeeper-policies`. This section provides the core of the landing zone functionality with Logging, Org Policies, and Shared Infrastructure like DNS and a Shared VPC (coming soon) as well as compliance enforcement in Config Controller.
```

Comment:
typo packages

Reply:
@davelanglois-ssc to smooth out the onboarding experience. We've gone through a few setups recently with clients and this is a stumbling block and slows things down. It's a bit more work on our end to keep the docs up and running but lets users simply run the command without having to bounce between tabs to find the correct versions.